tests/integ/test_track_from_processing_job.py

```diff
@@ -20,12 +20,12 @@
 
 @pytest.mark.slow
 def test_track_from_processing_job(sagemaker_boto_client, processing_job_name):
-
+    processing_job_name = "smexperiments-integ-549de818-b4bd-42d0-8ce7-1cb4cb3573d9"
```

Comment:
meant to have this hard coded job name here?

Reply:
Yeah, this is used to fix the slow tests similar to our test_track_from_training_job